sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/Helpers/RecordedDelegatingHandler.cs

```diff
@@ -0,0 +1,97 @@
+// ------------------------------------------------------------------------------------------------
+// <copyright file="RecordedDelegatingHandler.cs" company="Microsoft Corporation">
+//   Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace DnsResolver.Tests.Helpers
+{
+    using System;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public class RecordedDelegatingHandler : DelegatingHandler
+    {
+        private readonly HttpResponseMessage _response;
```

Comment:
Is this `_varName` a convention in the SDK repo for private fields?

Reply:
Yes, we used this in private dns.